chingsley/ireporter-react
# 164040787 Enable users to make a report

#### What does this PR do?
It implements the report creating feature enabling users to make a report

#### Description of Task to be completed?
Logged in users should be able to make a report by providing the coordinates of the location, the address and then some comments.

#### How should this be manually tested?
- Visit the app URL `https://chingsley-ireporter-react.herokuapp.com/create_report `
- Provide the necessary information as requested on the form
- hit the send button
- Note that the coordinates, address and comments are required fields

#### Any background context you want to provide?
- N/A

#### What are the relevant pivotal tracker stories?
https://www.pivotaltracker.com/story/show/164040787

#### Questions:
N/A
